tests/packages/Microsoft.Bot.Components.Telephony.Tests/Integration/CallTransferTests/CallTransfer_HappyPath.test.dialog

```diff
@@ -2,17 +2,21 @@
     "$schema": "../../../tests.schema",
     "$kind": "Microsoft.Test.Script",
     "dialog": "CallTransfer_BaseScenario.test",
-    "script": [
-        {
-            "$kind": "Microsoft.Test.UserSays",
-            "text": "Hello I'm Calculon"
-        },
-        {
-            "$kind": "Microsoft.Test.AssertReplyActivity",
-            "assertions": [
-                "type == 'event'",
-                "value.TargetPhoneNumber == '+1555443443'"
-            ]
-        }
-    ]
+  "script": [
+    {
+      "$kind": "Microsoft.Test.UserSays",
+      "text": "Hello I'm Calculon"
+    },
+    {
+      "$kind": "Microsoft.Test.AssertReplyActivity",
+      "assertions": [
+        "type == 'event'",
+        "value.TargetPhoneNumber == '+15554434432'"
+      ]
+    },
+    {
+      "$kind": "Microsoft.Test.AssertReply",
```

Comment:
I know we went back and forth on this but this activity seems to conflict with the spec. We say no activity / action executes after call transfer but I can see that its hard to simulate in a mock. Do you know if we can add a comment to explain that this?